bigquery/tests/integration.sh

```diff
@@ -44,13 +47,34 @@ sleep 5
 expr $(curl -s 'http://localhost:9200/1000_genomes/_search' | jq -r '.hits.total') = "3500"
```

Comment:
This line doesn't work. There are 3714 documents, but the test still passes. I was going to fix, but since you're changing this file, can you fix? A normal "if statement" might be most readable.

Reply:
Huh good point...not sure why expr wasn't working but just switched to an if like the others.